### DISCUSSION TITLE
[selfhosting][proxy] 500 Not Found after login

### DISCUSSION THREAD

**Discussion by mlawren** [PostOwl/postowl#45] [Help]
I'm attempting to run PostOwl locally, proxied behind lighttpd. After submitting the login form PostOwl responds with a `500 Not Found` error and a link to "Start over" at the root url. Has anyone run this configuration before and have some hints?

**Comment by keybits** [comment]
Thanks for reporting this,

Is this one different from https://github.com/PostOwl/postowl/discussions/44? The 500 error is expected if the password is incorrect. But perhaps it's related to the `ORIGIN` issue being discussed in the other thread?

**Comment by mlawren** [comment]
This was definately a separate issue to #44. My original description is incorrect: For this issue I am in fact querying the dev server directly, not behind a proxy (to avoid the whole cross-site POST error).

But it turns out my issue was wrong password (fat fingered in the `.env` file), so this discussion can stop. Thanks for the quick response.

p.s.  I'm not sure that generating a 500 error on wrong password is very helpful. In fact, "downright misleading" kind of springs to mind :) Perhaps a 401 or 403?

**Reply by keybits** [reply]
Thanks for the update. Agreed on the unhelpful error - I've opened: https://github.com/PostOwl/postowl/issues/47

**Comment by keybits** [comment]
This is now fixed with friendly error message if an incorrect password is entered! https://github.com/PostOwl/postowl/commit/489483a82ef1ca9a9fa185dcbd570d879eb7476f

**Comment by mlawren** [comment]
Thanks for the update!

-- 
Mark Lawrence
